### ISSUE TITLE
Movies or dirs. containing "PT" in name cannot be handled correctly.

### ISSUE THREAD

**Issue by butty2017** [Komet/MediaElch#1116]
**Describe the bug**
There are two problems:
1. If some number of movies' parent directory's name contains "PT"(case insensitive), the movies are treated as one movie.
  - Eg.: "/home/uname/Videos/PT2-2/*.mp4" are treated as one movie having first one's title.
2. If movie's name contains "PT"(case insensitive), the movie is ignored.
  - Eg.: "/home/uname/misc/Videos/test-movies/test/PT1-test.vob" is not shown in movie list.

It seems that "PT" is treated as a keyword "part" in a series of movies.

**To Reproduce**
1. Start MediaElch.
2. Scan movies having above dir/movie names.
3. See the movie list.

**Expected behavior**
Directories or movies containing "PT"(case insensitive) are treated as-is.

**MediaElch Version:**
 - [X] 2.8.3-dev_2020-12-20 (nightly)
 - [X] 2.8.2 stable

**Operating System:**
 - Linux (distribution: Linux Mint 20)

Thanks in advance.

**Comment by bugwelle** [comment]
Hi,

sorry, I forgot about this issue (it was around Christmas when you posted it).

Do you have `separate folders" enabled in your settings for your movies directory?

**Comment by butty2017** [comment]
Hi, 

Ok, no problem.

I am disabling "separate folders", however, enabling it will not change the problem, I remember.

**Comment by bugwelle** [comment]
Ok. I'll try to reproduce it:


1.

```sh
cd Videos
mkdir "PT2"
touch "PT2/Iron Man.mov"
touch "PT2/Harry Potter.mov"
```

Add `Videos` to MediaElch with "Sep. Folders" __disabled__.
I did the same with a the directory called `PT2-2`. I could _not_ reproduce your issue:

<img width="1110" alt="Screen Shot 2021-02-26 at 13 54 55" src="https://user-images.githubusercontent.com/1667306/109302930-479f7180-783a-11eb-812b-c881d72543b5.png">

2. 


```sh
mkdir -p test-movies/test
touch test-movies/test/PT1-test.vob
mkdir "test-movies/Iron Man/"
touch "test-movies/Iron Man/PT1-test-Iron-Man.vob"
```

I could not reproduce this issue as well.

------------

It may be that your issue was fixed by https://github.com/Komet/MediaElch/pull/1197 
I fixed an issue with handling of "pt" there as well.

That is also part of v2.8.6.

Did you try it with a more recent version of MediaElch again? If yes, could you give me a more detailed directory structure? :)

**Comment by butty2017** [comment]
Hi, 
Thanks for checking and reply. Yes, the problem is surely fixed in v2.8.6. I suppose the problem  was caused by #1197.

I am using latest version, however, couldn't aware of the fix because I had made a workaround in video dirs. for the problem. Today, I tested both with v2.8.0 and v2.8.6 and found v2.8.6 has no problem.

Thank you very much!

**Comment by bugwelle** [comment]
Thank you for verifying that it works! 

I'm always thankful for people reporting bug-reports. :-)
